mitodl/mit-open
# ETL for Open Learning events via API

### What are the relevant tickets?
Closes https://github.com/mitodl/mit-open/issues/681

### Description (What does it do?)
Refactors the Open Learning events ETL pipeline to use API calls instead of scraping HTML from the events page.


### How can this be tested?
Run `./manage.py backpopulate_news_events --delete` if you already have some news/event items.
Run `./manage.py backpopulate_news_events -p ol_events_etl`

It should complete successfully and create 1 `FeedSource` and about 3 `FeedItems` of type `events`, with populated image, audience, location, and event_type fields.  All the event dates should be today or later, no past events.

You can view them in django admin or http://localhost:8063/api/v0/news_events/
